analytics_data_api/v0/views/users.py

```diff
@@ -0,0 +1,75 @@
+"""
+API methods for user data.
+"""
+from django.http import Http404
+from rest_framework import generics
+
+from analytics_data_api.v0.models import UserProfile
+from analytics_data_api.v0.serializers import UserProfileSerializer
+
+
+class UserListView(generics.ListAPIView):
+    """
+    Get the (paginated) list of all users (students and staff).
```

Comment:
@bradenmacdonald I'm not familiar with rest_framework... Are `ListAPIView` responses paginated by default?